src/Reflection/ReflectionClass.php

```diff
@@ -62,6 +62,8 @@
 
 class ReflectionClass implements Reflection
 {
+    public const IS_READONLY = 65536;
```

Comment:
Should we make this `public`? Seems like a mistake.

Reply:
It's public because of tests: https://github.com/Roave/BetterReflection/pull/1192/files#diff-e0dd711174c42542396ea4773a4b7db9ba6025250e7d50950c152a6d43b5fe91R1154

It's same as https://github.com/Roave/BetterReflection/blob/5.10.x/src/Reflection/ReflectionClassConstant.php#L21

Maybe I can mark it as `@internal` ?

Reply:
`@internal` with an explanation would suffice 👍